### ISSUE TITLE
Checkpoints of the model

### ISSUE THREAD

**Issue by Sassanmtr** [harryzhangOG/flowbotpp#1]
Hi there, thank you for sharing the code for this interesting project. I'm planning to use your work as a baseline in my own project. I wanted to ask if there's a possibility of sharing the checkpoint of your model?

**Comment by Jade0716** [comment]
Hi,did you use this work as a baseline in your project?

**Comment by Sassanmtr** [comment]
No, I didn't have access to the checkpoints. I used UMPNet instead:
"https://arxiv.org/pdf/2404.14968"
